apps/betterangels-backend/accounts/schema.py

```diff
@@ -76,6 +77,27 @@ def generate_magic_link(self, info: Info, data: MagicLinkInput) -> MagicLinkResp
         send_magic_link(data.email, base_url)
         return MagicLinkResponse(message="Email link sent.")
 
+    @strawberry_django.mutation(permission_classes=[IsAuthenticated])
+    def update_current_user(self, info: Info, data: UpdateUserInput) -> UserType:
+        current_user = get_current_user(info)
```

Comment:
i believe this already brings back the user you're fetching on line 88, please verify
```suggestion
        user = get_current_user(info)
```

Reply:
🤦‍♂️ peak 2am code. thanks for catching